doc/howto/cluster_recover.md

```diff
@@ -6,43 +6,45 @@ In that case, no operations are possible on this member, and neither are operati
 See {ref}`clustering-offline-members` and {ref}`cluster-automatic-evacuation` for more information.
 
 If you can bring the offline cluster members back or delete them from the cluster, operation resumes as normal.
-If this is not possible, there are a few ways to recover the cluster, depending on the scenario that caused the failure.
-See the following sections for details.
+If the cluster members are lost permanently (e.g. disk failure), it is possible to recover them if at least one member survives.
```

Comment:
What is "them" in this context? If the members are lost permanently, how can we recover them?

Reply:
These paragraphs didn't make a lot of sense altogether; I've made some changes for clarity.